src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -138,60 +149,107 @@ const TableHeader: React.FC<TableHeaderProps> = ({
-            onClick={leaveSeat}
-          >
-            <FontAwesomeIcon
-              className="back-button-icon"
-              icon={faChevronLeft}
-              size="xs"
-            />
-            Leave table
-          </button>
-        </div>
-        <div className="table-title-container">
-          <div className="private-table-title">
-            {tableTitle}
-
-            {tableCapacity && (
-              <span className="private-table-seats-left">
-                {numberOfSeatsLeft} seats left
-              </span>
-            )}
-          </div>
-          {tableOfUser && tableOfUser.subtitle && (
-            <div className="private-table-subtitle">{tableOfUser.subtitle}</div>
-          )}
-        </div>
-        <div className="lock-button-container">
+    <div className="row table-header">
+      <div className="table-header__leave-table">
+        <button
+          type="button"
+          title={"Leave " + seatedAtTable}
+          className="table-header__leave-button"
+          id="leave-seat"
+          onClick={leaveSeat}
+        >
           <FontAwesomeIcon
-            className="lock-table-icon"
-            icon={isCurrentTableLocked ? faLock : faLockOpen}
-            size="sm"
+            className="table-header__leave-button--icon"
+            icon={faChevronLeft}
+            size="xs"
           />
-          <div className="lock-table-checbox-indication">
-            {isCurrentTableLocked ? "Table Locked" : "Lock Table"}
+          Leave table
+        </button>
+      </div>
+
+      <div className="table-header__topic-info">
+        <div className="row table-header__topic">
+          <div>{tableTitle}</div>
+
+          <div className="table-header__edit-topic-button" onClick={show}>
+            <FontAwesomeIcon icon={faPen} />
           </div>
-          <label className="switch">
-            <input
-              type="checkbox"
-              className="switch-hidden-input"
-              checked={!!isCurrentTableLocked}
-              onChange={toggleIsCurrentTableLocked}
-            />
-            <span className="slider" />
-          </label>
         </div>
+
+        {tableCapacity && (
+          <span className="table-header__seats-left">
+            {numberOfSeatsLeft} seats left
+          </span>
+        )}
+
+        {tableOfUser && tableOfUser.subtitle && (
+          <label>{tableOfUser.subtitle}</label>
+        )}
+      </div>
+
+      <div className="table-header__lock-button">
+        <FontAwesomeIcon
+          className="table-header__lock-button--icon"
+          icon={isCurrentTableLocked ? faLock : faLockOpen}
+          size="sm"
+        />
+        <div className="table-header__lock-indication">
+          {isCurrentTableLocked ? "Table Locked" : "Lock Table"}
+        </div>
+        <label className="switch table-header__lock-toggle">
+          <input
+            type="checkbox"
+            className="switch-hidden-input"
```

Comment:
We have a generic/standard toggle component that can be used here now.

I forget what it's called off the top of my head though.. because it's not `Toggle` like I would have expected.

Reply:
Seems it's `Checkbox` with the `toggler` prop set to `true`:

- https://github.com/sparkletown/sparkle/blob/staging/src/components/atoms/Checkbox/Checkbox.tsx

Sidenote: It would be good to make a `Switch` or `Toggler` or similar named component that wraps this one, as `Checkbox` isn't what I would expect/search for when looking for this, it's more of an implementation detail (cc // @mike-lvov FYI)

Reply:
Added `Toggler` component in: https://github.com/sparkletown/sparkle/pull/1377/commits/de659957079dd74136f26d29e9b952be12add510

Reply:
@denisdimitrov I think what was suggested is to reuse my `Checkbox` component
https://github.com/sparkletown/sparkle/blob/staging/src/components/atoms/Checkbox/Checkbox.tsx.

Reply:
Ya, like making a tiny wrapper around `Checkbox` called `Toggler` that sets that mode on it.

Reply:
@0xdevalias @mike-lvov 
I understand your suggestion but atoms are supposed to be independent. If someone removes `Checkbox` tomorrow. He shouldn't go to another atom component and 'fix' it. If I use `Checkbox` for the `Toggler` component I'd have to make it an organism instead of an atom, that way it'd be correct. I don't really mind doing it, but `Toggler` feels like a small, common component which should be an atom.

Reply:
I don't know enough about that methodology to say one way or another on the specifics of that. But reusing existing code is better than duplicating code.

Personally I don't feel like a tiny wrapper like that should break the 'atom' concept, but.. I don't really understand it in any depth.

(tbh I'm not sure I even like it as a design methodology, at least not in a strict sense. but it's what we have right now..)

Reply:
Sounds good, since you both agree on this approach. I'm implementing it that way. Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a